app/assets/javascripts/code/services/_mobile_sessions.js

```diff
@@ -95,12 +98,19 @@ export const mobileSessions = (
 
     reSelectSession: function(id) {
```

Comment:
Not sure what this is doing and if I should clear the starting points here like I do in selectSession